### ISSUE TITLE
Training and Validation Losses

### ISSUE THREAD

**Issue by nikosphys** [FlorentF9/DeepTemporalClustering#16]
Can you please explain the losses: L, Lc, Lr and T

**Comment by FlorentF9** [comment]
Lc is the clustering loss (soft hardening loss using KL divergence on the soft assignments), Lr is the autoencoder reconstruction loss (MSE) and L = Lr + gamma * Lc is the total loss, where gamma is a weighting hyperparameter. T stands for nothing and should be removed.
